code/game/objects/items/storage/toolbox.dm

```diff
@@ -94,6 +94,40 @@ GLOBAL_LIST_EMPTY(rubber_toolbox_icons)
 /obj/item/storage/toolbox/mechanical/old/heirloom/PopulateContents()
 	return
 
+
+/obj/item/storage/toolbox/mechanical/old/clean
+	name = "toolbox"
+	desc = "A old, blue toolbox, it looks robust."

```

Comment:
An old

Reply:
Please either mark these as Kepler changes or put them into the modular folder